cypress/e2e/dashboard.cy.ts

```diff
@@ -0,0 +1,205 @@
+    login(keys.unfundedSecretKey);
+    cy.get('[data-cy="public-key-value"]').should(
+      "have.text",
+      keys.unfundedPublicKey
+    );
+  });
+
+  it("Should show the correct balance for a funded account", () => {
+    login(keys.fundedSecretKey);
+    cy.get('[data-cy="balance-value"]').should(
+      "have.text",
+      "10000.0000000 Lumens (XLM)"
+    );
+  });
+
+  it("Should show the correct balance for an unfunded account", () => {
+    login(keys.unfundedSecretKey);
+    cy.get('[data-cy="balance-value"]').should("have.text", "0 Lumens (XLM)");
+  });
+
+  describe("Dashboard header", () => {
+    const abbreviatePublicKey: (key: string) => string = (
+      publicKey
+    ): string => {
+      return publicKey.slice(0, 5) + "..." + publicKey.slice(-2);
+    };
+
+    it("Should have the wallet brand", () => {
+      cy.visit("/dashboard");
+      cy.get('[data-cy="header-brand"]').should("have.text", "My_Wallet");
+    });
+
+    it("Should display the correct abbreviated public key when user logs in with a secret key", () => {
+      const abbreviatedPublicKey: string = abbreviatePublicKey(
+        keys.unfundedPublicKey
+      );
+      login(keys.unfundedSecretKey);
+      cy.get('[data-cy="abbreviated-public-key"]').should(
+        "have.text",
+        abbreviatedPublicKey
+      );
+    });
+
+    it("Should have a copy button with the text 'copy'", () => {
+      cy.visit("/dashboard");
+      cy.get('[data-cy="copy-button"]')
+        .should("exist")
+        .should("have.text", "copy");
+    });
+
+    it("Should have a log out button with the text 'log out'", () => {
+      cy.visit("/dashboard");
+      cy.get('[data-cy="log-out-button"]')
+        .should("exist")
+        .should("have.text", "Sign out");
+    });
+
+    describe("Copy button", () => {
+      it("Should change the text of the copy button to 'copied! when the user clicks on it", () => {
+        cy.visit("/dashboard");
+        cy.get('[data-cy="copy-button"]').should("exist").trigger("click");
+        cy.get('[data-cy="copy-button"]').should("have.text", "copied!");
+      });
+
+      it("Should copy the public key to the clipboard when the user clicks on the copy button", () => {
+        login(keys.fundedSecretKey);
+        cy.get('[data-cy="copy-button"]').should("exist").trigger("click");
+        cy.window().then((win) => {
+          win.navigator.clipboard.readText().then((text) => {
+            expect(text).to.eq(keys.fundedPublicKey);
+          });
+        });
+      });
+
+      it("Should change the text of the copy button back to 'copy' after 1 second", () => {
+        const delayTime: number = 1001;
+        cy.visit("/dashboard");
+        cy.get('[data-cy="copy-button"]').trigger("click");
+        cy.wait(delayTime);
+        cy.get('[data-cy="copy-button"]').should("have.text", "copy");
+      });
+    });
+
+    describe("Log out button", () => {
+      const homeUrl: string = Cypress.config().baseUrl || "";
```

Comment:
Why we're not using Cypress.env here?

Reply:
I have replaced that line and followed your recommendation.